_pages/general-information-and-resources/distributed.md

```diff
@@ -92,7 +88,6 @@ From Peter Karman and Becky Sweger:
 
 ## Keeping in touch
 
-- **Log into the weekly all-hands meeting a few minutes early.** It's a good chance to see faces and chat before the meeting starts.
 - **Over communicate.** Working remote means you can't rely on your physical presence to communicate to everyone else whether or not you're "at work." For 18F team members, use the [18F Out of Office (OOO) calendar](https://www.google.com/calendar/embed?src=gsa.gov_bth7useo0eeiicjgos2di6ph8k%40group.calendar.google.com&ctz=America/New_York) if you're assigned to an office and working remotely or if you're working in a different place than usual. On phone calls, don't be afraid to jump in and tell people if you can't hear.
```

Comment:
Maybe put this one back, but change it to something like `**Log into your team's regular meetings a few minutes early**`, so that it's applicable across TTS teams? What do you think? 18F and Solutions each have biweekly meetings, for example.